src/main/resources/static/css/style.css

```diff
@@ -2943,3 +2943,34 @@ input:checked+.form-slider:before {
 #saved {
 	display: none;
 }
+
+.animate
+{
+	transition: all 0.1s;
+	-webkit-transition: all 0.1s;
```

Comment:
I think the `-webkit` things can be omitted, since all modern browsers support these features (and we don't really care about IE11, do we).